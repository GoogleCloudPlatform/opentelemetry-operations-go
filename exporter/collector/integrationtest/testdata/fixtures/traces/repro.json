{
    "resourceSpans": [
      {
        "resource": {
          "attributes": [
            {
              "key": "host.name",
              "value": {
                "stringValue": "vx-event-watcher-7bdf7f657c-l65pb"
              }
            },
            {
              "key": "service.name",
              "value": {
                "stringValue": "model-start-up"
              }
            },
            {
              "key": "telemetry.sdk.language",
              "value": {
                "stringValue": "go"
              }
            },
            {
              "key": "telemetry.sdk.name",
              "value": {
                "stringValue": "opentelemetry"
              }
            },
            {
              "key": "telemetry.sdk.version",
              "value": {
                "stringValue": "0.19.0"
              }
            }
          ]
        },
        "scopeSpans": [
          {
            "scope": {
            },
            "spans": [
              {
                "traceId": "4fe6e2a7cd0de3a1076362c74d21f1b0",
                "spanId": "73ba0470cf8d6d02",
                "parentSpanId": "",
                "name": "test-small-gpt-20220425-9dff66f59-pslkw",
                "kind": "SPAN_KIND_INTERNAL",
                "startTimeUnixNano": "1662591502000000000", 
                "endTimeUnixNano": "1662591604837877541",
                "attributes": [
                  {
                    "key": "k8s.namespace.name",
                    "value": {
                      "stringValue": "default"
                    }
                  },
                  {
                    "key": "generation",
                    "value": {
                      "intValue": "0"
                    }
                  }
                ],
                "status": {}
              },
              {
                "traceId": "4fe6e2a7cd0de3a1076362c74d21f1b0",
                "spanId": "9d48e19f405a39f0",
                "parentSpanId": "73ba0470cf8d6d02",
                "name": "Scheduling",
                "kind": "SPAN_KIND_INTERNAL",
                "startTimeUnixNano": "1662591500000000000",
                "endTimeUnixNano": "1662591500000000000",
                "attributes": [
                    {
                      "key": "k8s.namespace.name",
                      "value": {
                        "stringValue": "default"
                      }
                    },
                    {
                      "key": "generation",
                      "value": {
                        "intValue": "0"
                      }
                    }
                ],
                "status": {
                    "code": "STATUS_CODE_OK",
                    "message": ""
                }
              },
              {
                "traceId": "4fe6e2a7cd0de3a1076362c74d21f1b0",
                "spanId": "9d48e19f405a39f0",
                "parentSpanId": "73ba0470cf8d6d02",
                "name": "WorkloadIdentityInit",
                "kind": "SPAN_KIND_INTERNAL",
                "startTimeUnixNano": "1662591502000000000",
                "endTimeUnixNano": "1662591565000000000",
                "attributes": [
                    {
                      "key": "k8s.namespace.name",
                      "value": {
                        "stringValue": "default"
                      }
                    },
                    {
                      "key": "generation",
                      "value": {
                        "intValue": "0"
                      }
                    }
                ],
                "status": {
                    "code": "STATUS_CODE_OK",
                    "message": ""
                }
              },
              {
                "traceId": "4fe6e2a7cd0de3a1076362c74d21f1b0",
                "spanId": "9d48e19f405a39f0",
                "parentSpanId": "73ba0470cf8d6d02",
                "name": "DownloadingModel",
                "kind": "SPAN_KIND_INTERNAL",
                "startTimeUnixNano": "1662591566000000000",
                "endTimeUnixNano": "1662591572000000000",
                "attributes": [
                    {
                      "key": "k8s.namespace.name",
                      "value": {
                        "stringValue": "default"
                      }
                    },
                    {
                      "key": "generation",
                      "value": {
                        "intValue": "0"
                      }
                    }
                ],
                "status": {
                    "code": "STATUS_CODE_OK",
                    "message": ""
                }
              },
              {
                "traceId": "4fe6e2a7cd0de3a1076362c74d21f1b0",
                "spanId": "9d48e19f405a39f0",
                "parentSpanId": "73ba0470cf8d6d02",
                "name": "DownloadingModel",
                "kind": "SPAN_KIND_INTERNAL",
                "startTimeUnixNano": "1662591566000000000",
                "endTimeUnixNano": "1662591572000000000",
                "attributes": [
                    {
                      "key": "k8s.namespace.name",
                      "value": {
                        "stringValue": "default"
                      }
                    },
                    {
                      "key": "generation",
                      "value": {
                        "intValue": "0"
                      }
                    }
                ],
                "status": {
                    "code": "STATUS_CODE_OK",
                    "message": ""
                }
              },
              {
                "traceId": "4fe6e2a7cd0de3a1076362c74d21f1b0",
                "spanId": "9d48e19f405a39f0",
                "parentSpanId": "73ba0470cf8d6d02",
                "name": "LoadingModel",
                "kind": "SPAN_KIND_INTERNAL",
                "startTimeUnixNano": "1662591580000000000",
                "endTimeUnixNano": "1662591604000000000",
                "attributes": [
                    {
                      "key": "k8s.namespace.name",
                      "value": {
                        "stringValue": "default"
                      }
                    },
                    {
                      "key": "generation",
                      "value": {
                        "intValue": "0"
                      }
                    }
                ],
                "status": {
                    "code": "STATUS_CODE_OK",
                    "message": ""
                }
              }
            ]
          }
        ]
      }
    ]
  }
  