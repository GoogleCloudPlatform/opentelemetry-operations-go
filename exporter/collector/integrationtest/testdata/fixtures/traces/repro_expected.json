{
  "batchWriteSpansRequest": [
    {
      "name": "projects/fakeprojectid",
      "spans": [
        {
          "name": "projects/fakeprojectid/traces/4fe6e2a7cd0de3a1076362c74d21f1b9/spans/73ba0470cf8d6d02",
          "spanId": "73ba0470cf8d6d02",
          "displayName": {
            "value": "test-small-gpt-20220425-9dff66f59-pslkw"
          },
          "startTime": "1970-01-01T00:00:00Z",
          "endTime": "1970-01-01T00:00:00Z",
          "attributes": {
            "attributeMap": {
              "g.co/agent": {
                "stringValue": {
                  "value": "opentelemetry-go 1.9.0; google-cloud-trace-exporter 1.8.8"
                }
              },
              "g.co/gae/app/module": {
                "stringValue": {
                  "value": "model-start-up"
                }
              },
              "g.co/r/generic_node/location": {
                "stringValue": {
                  "value": "global"
                }
              },
              "g.co/r/generic_node/namespace": {
                "stringValue": {}
              },
              "g.co/r/generic_node/node_id": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "generation": {
                "intValue": "0"
              },
              "host.name": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "k8s.namespace.name": {
                "stringValue": {
                  "value": "default"
                }
              },
              "otel.scope.name": {
                "stringValue": {}
              },
              "telemetry.sdk.language": {
                "stringValue": {
                  "value": "go"
                }
              },
              "telemetry.sdk.name": {
                "stringValue": {
                  "value": "opentelemetry"
                }
              },
              "telemetry.sdk.version": {
                "stringValue": {
                  "value": "0.19.0"
                }
              }
            }
          },
          "sameProcessAsParentSpan": true,
          "spanKind": "INTERNAL"
        },
        {
          "name": "projects/fakeprojectid/traces/4fe6e2a7cd0de3a1076362c74d21f1b9/spans/9d48e19f405a39f0",
          "spanId": "9d48e19f405a39f0",
          "parentSpanId": "73ba0470cf8d6d02",
          "displayName": {
            "value": "Scheduling"
          },
          "startTime": "1970-01-01T00:00:00Z",
          "endTime": "1970-01-01T00:00:00Z",
          "attributes": {
            "attributeMap": {
              "g.co/agent": {
                "stringValue": {
                  "value": "opentelemetry-go 1.9.0; google-cloud-trace-exporter 1.8.8"
                }
              },
              "g.co/gae/app/module": {
                "stringValue": {
                  "value": "model-start-up"
                }
              },
              "g.co/r/generic_node/location": {
                "stringValue": {
                  "value": "global"
                }
              },
              "g.co/r/generic_node/namespace": {
                "stringValue": {}
              },
              "g.co/r/generic_node/node_id": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "generation": {
                "intValue": "0"
              },
              "host.name": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "k8s.namespace.name": {
                "stringValue": {
                  "value": "default"
                }
              },
              "otel.scope.name": {
                "stringValue": {}
              },
              "telemetry.sdk.language": {
                "stringValue": {
                  "value": "go"
                }
              },
              "telemetry.sdk.name": {
                "stringValue": {
                  "value": "opentelemetry"
                }
              },
              "telemetry.sdk.version": {
                "stringValue": {
                  "value": "0.19.0"
                }
              }
            }
          },
          "status": {},
          "sameProcessAsParentSpan": true,
          "spanKind": "INTERNAL"
        },
        {
          "name": "projects/fakeprojectid/traces/4fe6e2a7cd0de3a1076362c74d21f1b9/spans/9d48e19f405a39f0",
          "spanId": "9d48e19f405a39f0",
          "parentSpanId": "73ba0470cf8d6d02",
          "displayName": {
            "value": "WorkloadIdentityInit"
          },
          "startTime": "1970-01-01T00:00:00Z",
          "endTime": "1970-01-01T00:00:00Z",
          "attributes": {
            "attributeMap": {
              "g.co/agent": {
                "stringValue": {
                  "value": "opentelemetry-go 1.9.0; google-cloud-trace-exporter 1.8.8"
                }
              },
              "g.co/gae/app/module": {
                "stringValue": {
                  "value": "model-start-up"
                }
              },
              "g.co/r/generic_node/location": {
                "stringValue": {
                  "value": "global"
                }
              },
              "g.co/r/generic_node/namespace": {
                "stringValue": {}
              },
              "g.co/r/generic_node/node_id": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "generation": {
                "intValue": "0"
              },
              "host.name": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "k8s.namespace.name": {
                "stringValue": {
                  "value": "default"
                }
              },
              "otel.scope.name": {
                "stringValue": {}
              },
              "telemetry.sdk.language": {
                "stringValue": {
                  "value": "go"
                }
              },
              "telemetry.sdk.name": {
                "stringValue": {
                  "value": "opentelemetry"
                }
              },
              "telemetry.sdk.version": {
                "stringValue": {
                  "value": "0.19.0"
                }
              }
            }
          },
          "status": {},
          "sameProcessAsParentSpan": true,
          "spanKind": "INTERNAL"
        },
        {
          "name": "projects/fakeprojectid/traces/4fe6e2a7cd0de3a1076362c74d21f1b9/spans/9d48e19f405a39f0",
          "spanId": "9d48e19f405a39f0",
          "parentSpanId": "73ba0470cf8d6d02",
          "displayName": {
            "value": "DownloadingModel"
          },
          "startTime": "1970-01-01T00:00:00Z",
          "endTime": "1970-01-01T00:00:00Z",
          "attributes": {
            "attributeMap": {
              "g.co/agent": {
                "stringValue": {
                  "value": "opentelemetry-go 1.9.0; google-cloud-trace-exporter 1.8.8"
                }
              },
              "g.co/gae/app/module": {
                "stringValue": {
                  "value": "model-start-up"
                }
              },
              "g.co/r/generic_node/location": {
                "stringValue": {
                  "value": "global"
                }
              },
              "g.co/r/generic_node/namespace": {
                "stringValue": {}
              },
              "g.co/r/generic_node/node_id": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "generation": {
                "intValue": "0"
              },
              "host.name": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "k8s.namespace.name": {
                "stringValue": {
                  "value": "default"
                }
              },
              "otel.scope.name": {
                "stringValue": {}
              },
              "telemetry.sdk.language": {
                "stringValue": {
                  "value": "go"
                }
              },
              "telemetry.sdk.name": {
                "stringValue": {
                  "value": "opentelemetry"
                }
              },
              "telemetry.sdk.version": {
                "stringValue": {
                  "value": "0.19.0"
                }
              }
            }
          },
          "status": {},
          "sameProcessAsParentSpan": true,
          "spanKind": "INTERNAL"
        },
        {
          "name": "projects/fakeprojectid/traces/4fe6e2a7cd0de3a1076362c74d21f1b9/spans/9d48e19f405a39f0",
          "spanId": "9d48e19f405a39f0",
          "parentSpanId": "73ba0470cf8d6d02",
          "displayName": {
            "value": "DownloadingModel"
          },
          "startTime": "1970-01-01T00:00:00Z",
          "endTime": "1970-01-01T00:00:00Z",
          "attributes": {
            "attributeMap": {
              "g.co/agent": {
                "stringValue": {
                  "value": "opentelemetry-go 1.9.0; google-cloud-trace-exporter 1.8.8"
                }
              },
              "g.co/gae/app/module": {
                "stringValue": {
                  "value": "model-start-up"
                }
              },
              "g.co/r/generic_node/location": {
                "stringValue": {
                  "value": "global"
                }
              },
              "g.co/r/generic_node/namespace": {
                "stringValue": {}
              },
              "g.co/r/generic_node/node_id": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "generation": {
                "intValue": "0"
              },
              "host.name": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "k8s.namespace.name": {
                "stringValue": {
                  "value": "default"
                }
              },
              "otel.scope.name": {
                "stringValue": {}
              },
              "telemetry.sdk.language": {
                "stringValue": {
                  "value": "go"
                }
              },
              "telemetry.sdk.name": {
                "stringValue": {
                  "value": "opentelemetry"
                }
              },
              "telemetry.sdk.version": {
                "stringValue": {
                  "value": "0.19.0"
                }
              }
            }
          },
          "status": {},
          "sameProcessAsParentSpan": true,
          "spanKind": "INTERNAL"
        },
        {
          "name": "projects/fakeprojectid/traces/4fe6e2a7cd0de3a1076362c74d21f1b9/spans/9d48e19f405a39f0",
          "spanId": "9d48e19f405a39f0",
          "parentSpanId": "73ba0470cf8d6d02",
          "displayName": {
            "value": "LoadingModel"
          },
          "startTime": "1970-01-01T00:00:00Z",
          "endTime": "1970-01-01T00:00:00Z",
          "attributes": {
            "attributeMap": {
              "g.co/agent": {
                "stringValue": {
                  "value": "opentelemetry-go 1.9.0; google-cloud-trace-exporter 1.8.8"
                }
              },
              "g.co/gae/app/module": {
                "stringValue": {
                  "value": "model-start-up"
                }
              },
              "g.co/r/generic_node/location": {
                "stringValue": {
                  "value": "global"
                }
              },
              "g.co/r/generic_node/namespace": {
                "stringValue": {}
              },
              "g.co/r/generic_node/node_id": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "generation": {
                "intValue": "0"
              },
              "host.name": {
                "stringValue": {
                  "value": "vx-event-watcher-7bdf7f657c-l65pb"
                }
              },
              "k8s.namespace.name": {
                "stringValue": {
                  "value": "default"
                }
              },
              "otel.scope.name": {
                "stringValue": {}
              },
              "telemetry.sdk.language": {
                "stringValue": {
                  "value": "go"
                }
              },
              "telemetry.sdk.name": {
                "stringValue": {
                  "value": "opentelemetry"
                }
              },
              "telemetry.sdk.version": {
                "stringValue": {
                  "value": "0.19.0"
                }
              }
            }
          },
          "status": {},
          "sameProcessAsParentSpan": true,
          "spanKind": "INTERNAL"
        }
      ]
    }
  ]
}
